pkg/test/framework/environments/kubernetes/app_test.go

```diff
@@ -12,7 +12,7 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-package cluster
+package kubernetes
```

Comment:
Elsewhere in Istio, we use `kube` for k8s-specific packages.  WDYT?

Reply:
I initially thought about doing it, but this class will access the other "kube" package as well that contains utility functions. It becomes a bit awkward when everything is named "kube". :)

Having said that, I don't mind it too much. I can change it if you have strong opinions on this.